src/admin/admin.resolver.ts

```diff
@@ -1,19 +1,20 @@
-import { Args, Int, Query, Resolver } from '@nestjs/graphql';
+import { UseGuards, UseInterceptors } from '@nestjs/common';
+import { Args, Context, Mutation, Resolver } from '@nestjs/graphql';
 
-import { AdminService } from './admin.service';
+import { LoginInput } from './dto';
 import { Admin } from './entities/admin.entity';
+import { LocalGqlAdminGuard } from './guards';
+import { SetCookieInterceptor } from './interceptors';
 
 @Resolver(() => Admin)
 export class AdminResolver {
-  constructor(private readonly adminService: AdminService) {}
-
-  @Query(() => [Admin], { name: 'admins' })
-  findAll() {
-    return this.adminService.findAll();
-  }
-
-  @Query(() => Admin, { name: 'admin' })
-  findOne(@Args('id', { type: () => Int }) id: number) {
-    return this.adminService.findOne(id);
+  @Mutation(() => Admin, { name: 'login', nullable: true })
+  @UseGuards(LocalGqlAdminGuard)
+  @UseInterceptors(
+    SetCookieInterceptor('is-logged-in', true, { httpOnly: false }),
```

Comment:
what about cookie expire time?

Reply:
there is no info about this feature, it's not mentioned

Reply:
maxAge property is a `CookieOptions` property which you can use to set expire time.

Reply:
I know, it's not mentioned, no need to be expired.

Reply:
nope!

Reply:
Let's have a 7-day auth cookie cuz it can be a security risk.

Reply:
done.